api.rst

```diff
@@ -1084,20 +1084,9 @@ Since it contains the ``films_id`` foreign key, it is possible to embed ``box_of
 
     curl "http://localhost:3000/box_office?select=bo_date,gross_revenue,films(title)&gross_revenue=gte.1000000"
 
-Embedding is also possible between ``box_office`` partitions and the ``films`` table:
-
-.. tabs::
-
-  .. code-tab:: http
-
-    GET /films?select=title,box_office_2021_02(bo_date,gross_revenue)&rating=gt.8 HTTP/1.1
-
-  .. code-tab:: bash Curl
-
-    curl "http://localhost:3000/films?select=title,box_office_2021_02(bo_date,gross_revenue)&rating=gt.8"
-
 .. note::
-  Partitioned tables can reference other tables since PostgreSQL 11 but can only be referenced from any other table since PostgreSQL 12.
+  * Partitioned tables can reference other tables since PostgreSQL 11 but can only be referenced from any other table since PostgreSQL 12.
+  * Partitions are not included in the :ref:`schema_cache` since they should not be exposed to such features. This also avoids embedding ambiguity for the parent partitioned table.
```

Comment:
```suggestion
  * Embedding between the partitioned table and its partitions is not allowed because it leads to ambiguity errors(see :ref:`embed_disamb`). :ref:`custom_queries` can be used if this is needed.
```

Reply:
Maybe it should be changed to something like "Embedding on partitions is not allowed because it leads to ambiguity errors between them and their parent partitioned table...". The ambiguity arises mostly when other table embeds the parent table.